session-signatures/getLitActionSessionSigs/README.md

```diff
@@ -0,0 +1,24 @@
+# Getting Session Signatures using the `getLitActionSessionSigs` Method
+## Understanding the Implementation
+1. Using an imported ethereum private key, connect the wallet to the Lit RPC endpoint `Chronicle Yellowstone`
+2. Connect to the Lit network using the LitNodeClient on the `datil-dev` network, specifying the local storage to hold the generated wallet and session keys
+3. Initialize a secure connection through the Lit login with LitAuthClient using the `LIT_RELAYER_API_KEY`
+4. Generate a wallet signature to use as an [Auth Method](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/types_src.AuthMethod.html)
+5. Use the Auth Method to mint a PKP through the LitAuthClient
+6. Get the session signatures for the newly minted PKP, specifying the ability to execute Lit Actions
+
+**NOTE**
+---
+
+You may notice that this is nearly identical to the `getPkpSessionSigs` implementation. This is because this function does get a PKP session signature, but it's purpose is to implement custom authentication/authorization within a Lit Action. An example of using `getLitActionSessionSigs` for custom authentication can be found [here](../../custom-auth/browser/src/index.ts#L284)
```

Comment:
I can't think of a good fix for this, but we probably shouldn't be linking directly to other guides with relative file paths. The links are too easy to break with us having no way of knowing that we need to update them (until a user reports it's a broken link)

Reply:
Would linking the master branch of the be fine then? Like:
https://github.com/LIT-Protocol/developer-guides-code/blob/master/custom-auth/browser/src/index.ts#L284

Reply:
Yea I think that's better than the relative path

Reply:
Changed to use a URL